docs/recommended-extensions.md

```diff
@@ -22,3 +22,4 @@ These extensions are not (yet?) defined in `.vscode/extensions.json`, but have b
 - [Markdown Extended](https://marketplace.visualstudio.com/items?itemName=jebbs.markdown-extended) (with `kbd` option disabled, `kbd` turns wiki-links into non-clickable buttons)
 - [GitDoc](https://marketplace.visualstudio.com/items?itemName=vsls-contrib.gitdoc) (easy version management via git auto commits)
 - [Markdown Footnotes](https://marketplace.visualstudio.com/items?itemName=bierner.markdown-footnotes) (Adds [^footnote] syntax support to VS Code's built-in markdown preview)
+- [Project Manager](https://marketplace.visualstudio.com/items?itemName=alefragnani.project-manager) Fast Switching for other Foam Repos. especially seperated repo, this extension is very useful and helful. More Easy Things is Making and Clone to Main Foam Only Folder (for example /Users/Form/Form)
```

Comment:
I think this would be better sorted as a recipe, what do you think?

What differences does this have with `Ctrl+R` (Open Folder)?

Reply:
Ctrl + R is Best thing in VS Code, but it's show all project in VSCode. Even not Foam Repo.
<img width="1244" alt="스크린샷 2021-05-15 오후 11 42 54" src="https://user-images.githubusercontent.com/1904967/118365338-4d098e00-b5d7-11eb-9334-fd0a39cca926.png"> arrange is better than Project Manger. this thing is only searching Foam Folders, not your Projects(working etc, ps. I'm not Developer & Programmer, but Ctrl + R is all the thing of open the VSCode thing. Prooject manager is only thing of Form Repo (If Arrange it), so Fastest switch is possible.

(ps. Sorry for Korean Language, Because, I'm Native Korean, and studying hard for Form. and i'm serve Several things in Github Project is maintained, but it's very mixing to Fast switching for this - like a Onyx string > my Pontoon strings > Joplin etc... it's very complexity thing. but this extension is only show form repo(if you setting foirm only folder things)

Reply:
Awesome!

I'd favour the description to be as clear as possible so just `Project Manager - Fast switching for multiple Foam repos` sounds better.

Reply:
I like this, but I also believe it belongs more to a recipe "How to manage multiple Foam projects" rather than the list of recommended extensions.
In the recommended extensions I see everything that is appropriate for a basic Foam setup.
Recipes cover more special cases (I am considering having multiple Foam workspaces and not being happy with the `ctrl+r` solution to be < 5% case)

thoughts?

Reply:
`recommand extensions` is my short think - this(project manager) is extension -> ah going to arrage recommand extension. until now, before read to your article, i don' think for `how to manage multiple Foam Project` in direct recipe. anyway, i wanna share for my form tip ;) it's tip(manage multiple Foam Project) is very useful for me

Reply:
well, so i thought, re-PR to Recipes.